contributors.yaml

```
@@ -119,4 +119,9 @@ contributors:
     given-names: "Matthew"
     affiliation: "University of Wisconsin-Madison"
     orcid: "https://orcid.org/0000-0003-4124-7862"
+
```

Fix missing dash (denotes that you're a separate person from the one above you).
```suggestion
  -
```